src/main/kotlin/org/elm/ide/inspections/ElmUnresolvedReferenceInspection.kt

```diff
@@ -50,10 +50,31 @@ class ElmUnresolvedReferenceInspection : ElmLocalInspection() {
             // contributed the reference.
             val errorRange = (element as? ElmTypeRef)?.upperCaseQID?.textRangeInParent
 
-            val description = "Unresolved reference '${ref.canonicalText}'"
             val fixes = mutableListOf<LocalQuickFix>()
-            if (AddImportFix.isAvailable(element)) fixes += AddImportFix()
-            if (AddQualifierFix.isAvailable(element)) fixes += AddQualifierFix()
+            val qualifierContext = AddQualifierFix.findApplicableContext(element)
+            val importContext = AddImportFix.findApplicableContext(element)
+            // Share the tracker between both fixes so that invoking one will hide the hint from the other
+            val invocationTracker = QuickFixInvocationTracker()
+
+            // Only show the hint for the qualifier fix if both are available.
```

Comment:
You can't show multiple hints at the same time. If we wanted to provide access to both actions in a hint, we would have to combine the quick fixes and show both actions in the dropdown. But this seems like the action that will be most common.